genon/preprocessor/facade/attachment_processor.py

```
@@ -1080,65 +1090,109 @@ async def __call__(self, request: Request, file_path: str, **kwargs: dict):
+            print(f"docling_result 저장 완료: {docling_dir}")
+
+        vectors_dir = result_dir / "vectors_result"
+        vectors_dir.mkdir(parents=True, exist_ok=True)
+        with open(vectors_dir / "vectors.json", "w", encoding="utf-8") as f:
+            if vectors and hasattr(vectors[0], 'model_dump'):
+                json.dump([v.model_dump() for v in vectors], f, ensure_ascii=False, indent=2)
+            else:
+                json.dump(vectors, f, ensure_ascii=False, indent=2)
+        print(f"vectors_result 저장 완료: {vectors_dir}")
+
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)